concepts/sako.scroll

```diff
@@ -14,15 +14,46 @@ originCommunity Polish Academy of Sciences
 reference https://historiainformatyki.pl/skan.php?doc_id=1489&type=pdf&for_download=1
 
 assignmentToken =
+lineCommentToken K)
```

Comment:
This is interesting! Why is the line comment token K)?

Reply:
I think this is influence from FORTRAN where lines starting with `C` are comments. In Polish `komentarz` means `comment`, so that probably is the reason for the `K`. As for the `)`, SAKO already has a label system with labels limiting themselves from the code by using the bracket, it might have been added for consistency. (Similarly to for loop delimiter `*)`)
Those are just my guesses, as I don't remember any explanation in the specification or articles.
PS. In English translation of SAKO used in English articles comments are started with `C)`.

Reply:
Neat! Thanks!